relationships/candidates/AWSELASTICACHEREDIS.yml

```diff
@@ -6,11 +6,11 @@ lookups:
       - domain: INFRA
         type: AWSELASTICACHEREDISNODE
     tags:
-      matchingMode: ALL
```

Comment:
Must be `ALL`, both endpoint and port must match: https://github.com/newrelic/entity-definitions/blob/main/docs/relationships/candidate_relationships.md#tags-mode

Reply:
@javimb  we will getting different tags names for cluster and node 
for cluster we will be getting : `aws.elasticache.configurationEndpointAddress` and `aws.elasticache.configurationEndpointPort`

for the node : `aws.elasticache.endpointAddress`, and `aws.elasticache.endpointPort`

Does the predicates will matches the tagkeys, if one doesn't exists

Reply:
Correct, the matching mode refers to the predicates. ALL predicates (endpoint and port) must match.

Reply:
BTW, for consistency with databases, this file should be called `REDIS.yml`.